src/App.js

```diff
@@ -1,23 +1,93 @@
-import logo from './logo.svg';
+
 import './App.css';
+import contactsArr from './contacts.json'
+import { useState } from 'react';
 
 function App() {
+
+  
+  
+  const fiveContacts = (contactsArr.slice(0,5));
+  console.log(fiveContacts)
+  const [contacts, setContacts] = useState(fiveContacts); 
+    
+    /* console.log(randomContact) */
+
+    const restOfContacts= contactsArr.slice(6)
+    const randomContact= restOfContacts[Math.floor(Math.random() * restOfContacts.length)]
+    console.log(randomContact)
+
+    const addRandomContact = () => {
+         const newContact = [...contacts, randomContact];
+    
+         setContacts(newContact);
+        
+      }
```

Comment:
You should have line 16,17 inside the addRandomContact() function -> to keep logic nicely organised. We shouldn't have loose lines of code lying around :)